### PR TITLE
Youtube viewer 3.5.8

### PR DESCRIPTION
**Pull request by zpuskas**

### CODE REVIEW

File: net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild]: Why not 7? Just update `gnome2_icon_cache_update` to use `xdg_icon_cache_update` from xdg-utils instead. 7 should work here.

File: net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	virtual/perl-Term-ReadLine
+	virtual/perl-Text-ParseWords
+	virtual/perl-Text-Tabs+Wrap
+	gtk? (
+		dev-perl/File-ShareDir
+		>=dev-perl/Gtk2-1.244.0
+		virtual/freedesktop-icon-theme
+		x11-libs/gdk-pixbuf:2[X,jpeg]
+	)
+	|| ( media-video/ffmpeg[openssl] media-video/ffmpeg[gnutls] )
```
Comment (juippis) [inline net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild]: No love for libressl? :)
(Im not asking you to test it, just if you are aware whether it works or not)

File: net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild (hunk 1 above)
Comment (zpuskas) [reply net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild]: Will try, seems straightforward.

File: net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild (hunk 2 above)
Comment (zpuskas) [reply net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild]: I've never tried libressl, I'll need to set up a Gentoo VM with libressl in it for this to be tested. I'll check, but resolving this issue might be out of scope for this PR.

Comment (juippis) [reply net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild]: I agree. As I said, I dont expect you to set up a libressl system for this one. Just if you were aware that it works/not from upstream issues, changelogs etc.

File: net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild (hunk 1 above)
Comment (zpuskas) [reply net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild]: Rebased and updated to EAPI 7 and xdg-utils.

File: net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild (hunk 2 above)
Comment (zpuskas) [reply net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild]: I'll check with upstream or will test it myself. I'll open a bug to myself in bugzilla to track this.

**Pull request closed without merging**